SwiftOverlays/SwiftOverlays.swift

```diff
@@ -245,6 +245,22 @@ open class SwiftOverlays: NSObject {
         
         return blocker
     }
+
+    /**
+     Shows *blocking* overlay *with progress view and text*,, centered in the app's main window
+     
+     - parameter text: Text to be shown on overlay
+     
+     - returns: Created overlay
+     */
+    open class func showBlockingProgressOverlay(_ text: String) -> UIProgressView  {
```

Comment:
I missed the updateOverlayProgress part. I will make an update to return a UIView and use updateOverlayProgress accordingly.